scripts/openzfs-exceptions.txt

```diff
@@ -86,10 +88,13 @@
 6418 131cc95
 6421 ca0bf58
 6434 472e7c6
+6459 - 			changes don't work properly with ZoL codebase
```

Comment:
These changes should work with ZoL.  However, it's entirely possible, even likely, that there are places in the ZoL code base which are Linux specified which need to be updated.